app/src/main/java/com/kickstarter/viewmodels/SettingsNewViewModel.kt

```diff
@@ -0,0 +1,16 @@
+package com.kickstarter.viewmodels
+
+import android.content.Context
+import android.databinding.BaseObservable
+import com.kickstarter.BuildConfig
+import com.kickstarter.libs.utils.ViewUtils
+
+class SettingsNewViewModel(private val context: Context) : BaseObservable() {
+
+    fun getVersion() = BuildConfig.VERSION_NAME
+
+    fun rateUsClick() {
+        ViewUtils.openStoreRating(context, context.packageName)
```

Comment:
I'm not sure what we gain by moving click logic from the activity to this new view model. The databinding is cool but is there a reason why this wouldn't live in the activity?

Reply:
It just keeps the activity much cleaner and we have a direct reference to the view. View binding dramatically reduces the number of times you have to refer to view names in your code.